### DISCUSSION TITLE
Question: RadioButtons SelectionChanged event, why not NewSelection and OldSelection property?

### DISCUSSION THREAD

**Discussion by marcelwgn** [microsoft/microsoft-ui-xaml#7511] [Q&A]
The RadioButtons SelectionChanged event comes with the standard SelectionChangedEventArgs which has a AddedItems and RemovedItems property. What was the reason to not use different event args, e.g. something similiar to the NavigationView selection changed args? Given that the selection can only remove one item at most and add one item at most, having lists here doesn't seem appropiate for the API.

**Comment by StephenLPeters** [comment]
Good point, unfortunately we've shipped the API and wont be able to change this until potentially Winui 4.

**Comment by marcelwgn** [comment]
Alrighty then, guess we will revisit this with WinUI 4 then.
